modules/ROOT/pages/parse-template-reference.adoc

```diff
@@ -51,26 +51,184 @@ Attributes of the `parse-template` element:
 
 
 [[code_example]]
-== Code Example
+== Examples
 
-The following example demonstrates the use of a Parse Template in an application
-that accepts queries by `employeeID` to acquire data about an employee.
+The examples use Parse Template to load data dynamically into HTML.
```

Comment:
Optional: Since the examples are being introduced here, should it be something like "These examples" ? 
Typically we don't use the definite article "the" until after 'the thing' has been introduced...

Reply:
The examples are in separate, longish subsections, so I didn't want them to expect to see them immediately but also didn't want to say "The examples in this section." I take your point but could go either way with "the" or "these".

Reply:
I added a bullet list with links to each example. And started with "The following examples..."